#158487964/Users should see a landing page with effective UI/UX

phemonick opened the pull request:
#### What does this PR do?
convert mockup to html , css and angularjs
#### Description of Task to be completed?
Users should be able to see an overview of what the game is about, how it's played and how it can be used to donate to charity and also see call to action buttons that lead them to signup or signin
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#158487964 Users should see a landing page with effective UI/UX
#### Screenshots (if appropriate)
<img width="1423" alt="screen shot 2018-07-09 at 6 55 03 pm" src="https://user-images.githubusercontent.com/23261181/42475743-7103ea88-83c3-11e8-882d-e23091ed016d.png">

#### Questions:

ozimos committed:
feat(endpoints): add search users and send mail endpoints

- add endpoint to send email
- add endpoint to search users

[Starts #158487963]

ozimos committed:
feat(max players): increase max player limit

[Features #158487963]

ozimos committed:
feat(notifications): add notifications for game play

- add notification modal for max players
- add invite player button

[Features #158487963]

ozimos committed:
feat(endpoints): refactor api response

[Features #158487963]

ozimos committed:
feat(endpoints): fix errors in client tests

- write client code in es5

[Features #158487963]

ozimos committed:
chore(client): add test for mailer endpoint

[Fixes #158487963]

ozimos committed:
chore(client): add delay for search input

[Finishes #158487963]

ozimos committed:
chore(client): fix failing test

[Finishes #158487963]

ozimos committed:
chore(client): add TTL feedback

[Finishes #158487963]

ozimos committed:
chore(client): add env variables to Heroku

[Finishes #158487963]

ozimos committed:
chore(client): fix hound issues

[Finishes #158487963]

ozimos committed:
chore(client): amend eslintrc rule

[Finishes #158487963]

Stylll reviewed (commented):
Nice work. However, I noticed there are no screenshots of the landing page in your PR request. You also have 3 console logs in the landingPage.js file. You should remove the console log messages and add a screenshot to your PR request.

ozimos reviewed (commented):
Good work,
However, your PR title does not follow our PR Naming convention. Please rename the PR.

otseobande committed:
feat(profile-image): add profile image upload on signup

- setup cloudinary and ng-file-upload
- seprated scss and css files and add adjust scss compile gulp task
- integrate image upload with signup page
- Fix right panel space bug on home page
- Change test browser to Chrome

[Finishes #158487979]

otseobande committed:
feat(profile-image): add profile image upload on signup

- setup cloudinary and ng-file-upload
- seprated scss and css files and add adjust scss compile gulp task
- integrate image upload with signup page
- Fix right panel space bug on home page
- Change test browser to Chrome

[Finishes #158487979]

Commit:
feature(landing-page): redesign landing page
- convert mockup to html
- implement authentication button feedback
- add profile images for team members
-resolve karma config error
[(finishes)#158487964]

Commit:
feature(landing-page): redesign landing page
- convert mockup to html
- implement authentication button feedback
- add profile images for team members
-resolve karma config error
[(finishes)#158487964]

Commit:
feature(landing-page): redesign landing page
- convert mockup to html
- implement authentication button feedback
- add profile images for team members
-resolve karma config error
[(finishes)#158487964]

Commit:
feature(landing-page): redesign landing page
- convert mockup to html
- implement authentication button feedback
- add profile images for team members
-resolve karma config error
[(finishes)#158487964]

Commit:
feature(landing-page): redesign landing page
- convert mockup to html
- implement authentication button feedback
- add profile images for team members
-resolve karma config error
[(finishes)#158487964]

ozimos committed:
chore(client): add reviewer feedback

[Finishes #158487963]

VictoriaOffoma commented on client/js/landingPage.js:
```diff
@@ -0,0 +1,6 @@
+window.addEventListener('load', () => {
+  let elems = document.querySelectorAll('.slider');
```
Is there a reason why we are using _let_ and not _const_  for the _elems_ variable?

VictoriaOffoma commented on client/js/landingPage.js:
```diff
@@ -0,0 +1,6 @@
+window.addEventListener('load', () => {
+  let elems = document.querySelectorAll('.slider');
+  let instances = M.Slider.init(elems, {
```
Is there a reason why we are using _let_ and not _const_ for the _instances_ variable?

VictoriaOffoma commented on client/css/landingpage.scss:
```diff
@@ -0,0 +1,363 @@
+    align-items: center;
+    justify-content: center;
+    font-size: 16px;
+    background: #006989;
```
do we have our colours declared in variables?

phemonick committed:
Merge branch 'staging' into feature/158487964/landing-page
